chacha20poly1305/src/lib.rs

```diff
@@ -29,7 +30,7 @@
 //!       174, 123, 232 ]
 //! );
 //! assert_eq!(
-//!     tag.as_ref(),
+//!     &tag.as_ref().declassify(),
```

Comment:
```suggestion
//!     tag.as_ref().declassify_ref(),
```
Otherwise I still get a failed build, now without the `check-secret-independence` feature.